src/app/components/Embeds/EmbedError/index.styles.tsx

```diff
@@ -13,16 +14,18 @@ const styles = {
       [mq.GROUP_4_MIN_WIDTH]: {
         margin: `0 0 ${spacings.TRIPLE}rem`,
       },
+      [mq.HIGH_CONTRAST]: {
+        border: `${pixelsToRem(3)}rem solid transparent`,
+      },
     }),
-  errorLinkWrapper: ({ fontVariants, spacings }: Theme) =>
+  errorLinkWrapper: ({ spacings }: Theme) =>
     css({
-      ...fontVariants.sansBold,
```

Comment:
I also removed this declaration because I didn't think it was doing anything? Can add back in if I'm mistaken.